environment-dev.yml

```diff
@@ -0,0 +1,9 @@
+name: dem
```

Comment:
@arthurduf Does ESA MAAP use `conda` as a package manager, for either the ADE or for the DPS or for both? I see that `sardem` is being installed via the Dockerfile ([link](https://github.com/MAAP-Project/get-dem/blob/3d42aba3fe93787528a8abe6a6389be215babf72/esa/Dockerfile#L8)), but I can't quite find where the environment.yaml file is being used.

Reply:
@nemo794, ESA doesn't need an `environment.yml` file because the only dependency other than `sardem` is `gdal`, and the Dockerfile is based on a `gdal` image:

```plain
FROM ghcr.io/osgeo/gdal:alpine-normal-3.8.2
```

All that would be required beyond that is to add `scalene` to the `pip install` with `sardem` once we get to the point of adding `scalene` for ESA.

Reply:
@nemo794 :  We use conda for the ADE but as @chuckwondo mentioned we have a different strategy for the application package.